Font_Awesome_Cookie_Info.md

```diff
@@ -0,0 +1,39 @@
+# FONT AWESOME COOKIE INFORMATION (DRAFT)
+
+## Classification Level
+Publicly available
+
+## Review Information
+
+__Mandatory Review Period__
+
+Yearly
+
+__Date of Last Review__
+
+
+
+## Introduction
+
+This cookie information document helps users identity what information we collect in cookie form and provides general 
+information about how and why we use cookies.
+
+## Cookie Usage Statement
+
+Usage of cookies aligns with our (privacy policy)[https://fontawesome.com/privacy]. Our general goal is to collect no more 
+data about a user than is necessary to provide our service and prevent abuse.
+
+## Cookies 
+
+| Cookie Key | Domain | Path | Cookie Classification | Cookie Type | Expiration | Description |
+|------------|--------|------|-----------------------|-------------|------------|-------------|
+| __cf_bm | .vimeo.com | / | Strictly Necessary | Third-party | 30 minutes | This cookie is used by Cloudflare to distinguish between humans and bots. |
```

Comment:
@alexpoiry are we sure `.vimeo.com` is the correct domain here?

Reply:
Nope, I think we are getting 3rd party vimeo cookies set because of our sweet sweet marketing vidoes.